src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,59 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
 public class WorkWithFile {
+    private static final int NAME_INDEX = 0;
+    private static final int PARAM_INDEX = 1;
+    private static final int RESULT_INDEX = 2;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        writeWtoFile(createReport(readFromFile(fromFileName)), toFileName);
```

Comment:
Split it into 3 lines to increase readability

Reply:
I divided the calls into three lines, but I had to create intermediate variables to transfer data between calls. I'm not sure it's good for the process.